crates/egui_kittest/src/builder.rs

```diff
@@ -78,13 +77,13 @@ impl<State> HarnessBuilder<State> {
     /// This sets up a [`WgpuTestRenderer`] with the default setup.
```

Comment:
Please fix this link 
```suggestion
    /// This sets up a [`crate::wgpu::WgpuTestRenderer`] with the default setup.
```

Reply:
or reimport it as you removed the import above